# Значение кватерниона Q1 [-1; 1]
float32 Q1

# Значение кватерниона Q2 [-1; 1]
float32 Q2

# Значение кватерниона Q3 [-1; 1]
float32 Q3

# Значение кватерниона Q4 [-1; 1]
float32 Q4
